identity-android/src/main/java/com/android/identity/android/util/NfcUtil.java

```diff
@@ -20,7 +28,7 @@
 import java.util.Arrays;
 import java.util.List;
 
-class NfcUtil {
+public class NfcUtil {
```

Comment:
do we want to add docstrings to functions in this files like these too (that are being made public)?